Week2/homework/js-exercises/Project_GradeCalculator.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+// Below is the function with the grade intervals defined according to the US grading system.
+function gradeCalculator(grade) {
+    if (grade>100) {
+        console.log('Are you kidding? You cannot have a grade of' + ' (' + grade + '%' + ')');
+    } else if (100>=grade>=90) {
+        console.log('You got an A' + ' (' + grade + '%' + ')');
```

Comment:
I think it is better to separate this logic using an && condition as it is more explicit and easier to understand. 🙂 

Otherwise, I think your logic is correct. But there are still some minor spacing/indentation issues which will be covered in future modules.